src/main/java/opendcs/util/functional/ResultSetConsumer.java

```diff
@@ -13,5 +15,5 @@ public interface ResultSetConsumer extends ThrowingConsumer<ResultSet,SQLExcepti
     /**
      * @param rs a valid resultset, user function should *NOT* call next
      */
-    public void accept(ResultSet rs) throws SQLException;
+    public void accept(ResultSet rs) throws SQLException, DatabaseException;

```

Comment:
Hmm, that does simplify some things, but not not entirely convinced that that's right approach... then again it might be the best approach until we get things better sorted out.